packages/llm/package.json

```diff
@@ -5,14 +5,18 @@
     "access": "public"
   },
   "version": "1.4.4",
-  "description": "Immutable & chainable LLM tools",
+  "description": "Type-safe LLM message handling with support for text, images, tool calls, and responses",
```

Comment:
Immutability and chainability are very important attributes of this lib. Make sure it appears in all the docs.